Applications/Games/Warcraft III Expansion Set/application.json

```diff
@@ -0,0 +1,4 @@
+{
+	"name": "Warcraft® III: Expansion Set",
+	"description": "The war rages on. The demonic threat has been banished from the battle-scarred fields of Azeroth, but peace is still a distant dream. The epic conflict that began in Warcraft III: Reign of Chaos continues with more units, more missions, and more explosive strategic combat."
```

Comment:
This is the description for TFT. Can you use a description which makes clear that the expansion set contains Reign of Chaos and TFT?

Reply:
Thanks for the quick feedback. I can't find any generic description of the two games, I've added a sentence that the set contains both games though.